index.html

```diff
@@ -1,28 +1,197 @@
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3"
+      crossorigin="anonymous"
+    />
     <script src="https://code.jquery.com/jquery-3.6.3.min.js"></script>
-    <link rel="stylesheet" href="style.css"/>
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css2family=Poppins:wght@400;500;600&display=swap">
-    <link href="https://fonts.googleapis.com/css2?family=Pathway+Gothic+One&display=swap" rel="stylesheet">
-    <script src="https://kit.fontawesome.com/3b9cd09213.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2family=Poppins:wght@400;500;600&display=swap"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Pathway+Gothic+One&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/3b9cd09213.js"
+      crossorigin="anonymous"
+    ></script>
     <title>JS Bootcamp</title>
-</head>
-<body>
-    
-
-
+  </head>
+  <body class="p-0">
+    <div class="header1 d-none d-md-flex w-100 justify-content-end">
+      <div class="social d-flex align-items-center">
+        <a href="https://www.facebook.com/"
+          ><i class="fab fa-facebook-f"></i
+        ></a>
+        <a href="https://twitter.com/"><i class="fab fa-twitter"></i></a>
+        <h4>English</h4>
+        <h4>My Page</h4>
+        <h4>Logout</h4>
+      </div>
+    </div>
+    <div class="header2 d-flex justify-content-between w-">
+      <div class="logo d-flex align-items-center w-1">
+        <a href="index.html"><img src="./images/logo1.png" alt="Logo" class="logo-img" /></a>
+      </div>
+      <nav>
+        <button
+          class="hamburger d-lg-none"
+          type="button"
+          aria-label="Toggle navigation"
+          aria-expanded="false"
+        >
+          <span></span>
+          <span></span>
+          <span></span>
+        </button>
+        <ul class="nav-links d-non d-md-flex">
+          <li><a href="index.html" class="scroll">Home</a></li>
+          <li><a href="#speaker" class="scroll">Speaker</a></li>
+          <li><a href="about.html" class="scroll">about</a></li>
+        </ul>
+      </nav>
+    </div>
 
+    <div class="present" id="home"> <section class="bg w-100">
+      <p class=" slt">"Hello! Sharing world"</p>
+      <h1 class="top">Welcome to the best JavaScript BootCamp</h1>
+      <p class="info border-2">
+        Unlock your potential as a web developer with our comprehensive
+        JavaScript bootcamp. From novice to expert, we'll guide you through the
+        ins and outs of the language and give you the skills you need to create
+        dynamic and interactive web applications.
+      </p>
+      <h2>2023.04.01 (THU) ~ 05.20 (FRI)</h2></section>
+    </div>
```

Comment:
- [ ] Great work working on the major changes requested. There are a few aspects you can improve on to make your design more appealing and resemble the original design

     - You can give your headline text a background image and 
          use the `background-clip` CSS property to position the 
          image within the text
     - You can reduce the font size of the paragraph text that has a brief description of the event and increase the padding between the text and the border
     
![image](https://user-images.githubusercontent.com/98453979/225682626-4d1b1fe0-5fdf-43a2-bc51-dad38e41ccd5.png)

Reply:
I got it @frank1738